### PR TITLE
Allows security_groups to update

### PR DESCRIPTION
**Pull request by gildub**
Adds update call to request and corresponding model

### CODE REVIEW

File: lib/fog/network/openstack/models/security_group.rb [hunk 1]
```diff
@@ -22,9 +22,14 @@ def security_group_rules
         end
 
         def save
-          raise Fog::Errors::Error, 'Resaving an existing object may create a duplicate' if persisted?
           merge_attributes(service.create_security_group(attributes).body['security_group'])
-          true
+          self
+        end
+
+        def update
```
Comment (Ladas) [inline lib/fog/network/openstack/models/security_group.rb]: maybe we could refactor this later to provider the standard interface of the base class create/update and save methods

**Comment by Ladas** [comment]
the, the hound is not deleting the non valid comments

**Comment by Ladas** [comment]
@gildub thanks :-)

File: lib/fog/network/openstack/models/security_group.rb (hunk 1 above)
Comment (gildub) [inline lib/fog/network/openstack/models/security_group.rb]: :+1:  Yes, I thought about that too. In my todo list...

**Pull request merged**